### ISSUE TITLE
Impala needs concat tests

### ISSUE THREAD

**Issue by chrisknoll** [OHDSI/SqlRender#58]
Due to an error we're seeing over at OHDSI/Achilles#191, this is due to some concatenation changes that were made for Impala, but there's no test cases for translating concatenation for impala.  We will need corresponding tests for Oracle if we are going to support mssql CONCAT() translation into oracle.

**Comment by schuemie** [comment]
Oracle CONCAT test here: https://github.com/OHDSI/SqlRender/blob/master/tests/testthat/test-translateSql.R#L566

What exactly was the problem with Impala?

**Comment by chrisknoll** [comment]
Hi, @schuemie , it's been a few months since I opened this issue, but I think I recall this issue was opened as a result of finding issues in OHDSI/Achilles#191, which was that concat() doesn't work the same across dbms (oracle lets you concat exactly 2 strings, postgresql and mssql lets you have N number of concats).  I thought I saw that concat on Impala had the same method signature, but that's not the case: Impala takes a variable number of args to concat like sql server:
https://www.cloudera.com/documentation/enterprise/5-9-x/topics/impala_string_functions.html

So, since it'st he case that Impala doesn't need translation for CONCAT (and I verified in the replacementPatterns.csv that there's no concat translation for impala) no test case is necessary here.

I'll close this. Thanks for following up on this.

-Chris
